packages/charm/src/manager.ts

```diff
@@ -1098,6 +1099,104 @@ export class CharmManager {
     await this.runtime.idle();
     await this.synced();
   }
+
+  /**
+   * Get the favorites cell from the home space (singleton across all spaces)
+   * @private
+   */
+  private getHomeFavorites(): Cell<Cell<unknown>[]> {
```

Comment:
Should these live elsewhere? They feel "Charm Manager"-y to me, but could have a new favorites module?